invertedai/cosimulation.py

```diff
@@ -23,36 +24,50 @@ class BasicCosimulation:
         used in arguments to :func:`initialize`.
     :param monitor_infraction: Whether to monitor driving infractions, at a small increase
         in latency and payload size.
-    :param render_birdview: Whether to render the bird's eye view of the simulation state
+    :param get_birdview: Whether to render the bird's eye view of the simulation state
         at each time step. It drastically increases the payload received from Inverted AI
         servers and therefore slows down the simulation - use only for debugging.
+    :param random_seed: Controls the stochastic aspects of simulation for reproducibility.
     """
 
     def __init__(
         self,
         location: str,
         ego_agent_mask: Optional[List[bool]] = None,
         monitor_infractions: bool = False,
-        render_birdview: bool = False,
+        get_birdview: bool = False,
+        random_seed: Optional[int] = None,
         # sufficient arguments to initialize must also be included
         **kwargs,
     ):
         self._location = location
-        response = initialize(location=location, **kwargs)
+        self.rng = random.Random(random_seed)
```

Comment:
If the seed is not given, I would prefer not to use a generator at all in the client, lest the customer gets the impression that they're somehow supposed to provide seed to the API calls.